tools/validators/instance_validator/tests/bad_building_type.yaml

```diff
@@ -0,0 +1,4 @@
+# Building
+UK-LON-S3:
+  type: FACILITIES/BUILDING/ERRORORO
```

Comment:
start integrating those files in your pr as unit tests

Reply:
Done, formatted these for use in future unit tests, once the ontology component of the validator is merged